tsconfig.build.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "exclude": [
-    "__test__",
+    "**/__test__/**",
```

Comment:
Nope, features isn't there. In fact we may not actually need that since `features` isn't inside `src`…?

With no `**` it just applies to top level directories, but we have `__test__` at different depths under different folders in `src`.

Reply:
I had to add `features` because of the typescript fixtures — the TS compiler complained that it was outside the `rootDir`, so it was trying to compile those fixtures with this TS config

That makes sense, I must have forgotten that `features` was top-level when I added it & changed `__test__`